waku/waku_api/rest/filter/handlers.nim

```diff
@@ -325,12 +382,15 @@ proc installFilterGetMessagesHandler(router: var RestRouter,
 
     return resp.get()
 
-proc installFilterRestApiHandlers*(router: var RestRouter,
-                                   node: WakuNode,
-                                   cache: MessageCache) =
-  installFilterPingSubscriberHandler(router, node)
-  installFilterPostSubscriptionsHandler(router, node, cache)
-  installFilterPutSubscriptionsHandler(router, node, cache)
-  installFilterDeleteSubscriptionsHandler(router, node, cache)
-  installFilterDeleteAllSubscriptionsHandler(router, node, cache)
+proc installFilterRestApiHandlers*(
+  router: var RestRouter,
+  node: WakuNode,
+  cache: MessageCache,
+  discHandler: Option[DiscoveryHandler] = none(DiscoveryHandler),
+  ) =
+  installFilterPingSubscriberHandler(router, node, discHandler)
+  installFilterPostSubscriptionsHandler(router, node, cache, discHandler)
+  installFilterPutSubscriptionsHandler(router, node, cache, discHandler)
+  installFilterDeleteSubscriptionsHandler(router, node, cache, discHandler)
+  installFilterDeleteAllSubscriptionsHandler(router, node, cache, discHandler)
   installFilterGetMessagesHandler(router, node, cache)
```

Comment:
Why this isn't receiving a `discHandler` too ?

Reply:
`GetMessages` is a local only command. It checks the local cache only.